### ISSUE TITLE
Use of italics doesn't align with VA style guide, remove from rich text menu

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#8034]
## Description
From the [VA style guide](https://design.va.gov/content-style-guide/bold-text), "Note: Don’t use italics. It doesn’t show up well on a screen, and sometimes it can be harder to read than normal text."

From a [Slack thread](https://dsva.slack.com/archives/C01RAS1KAQK/p1644597354096579), Martha Wilkes brought up the problems with the use of italics.

Given these things, it may be recommended to remove italics from the rich text menu for editors.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review

### CMS Team
Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [x] `⭐️ Product support`
  - [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@EWashb This is something that I was curious about - if editors shouldn't be using it why is it in the editor - but I didn't realize we had a ticket for.

What are your thoughts on this? Do you think it could be a possibility to bring up? We are updating the CKEditor in D10, technically none of the functionality would change but could we consider changing it?

**Comment by EWashb** [comment]
@laflannery I have no clue why we still allow it. I don't know any instance where it would be appropriate for an editor to use Italics. We can definitely consider changing the available options for editors and removing that. We could tie it to the Drupal 10 upgrade since there may be additional upgrades we need to do to that. I'll add this blocked by the Ckeditor5 work.

**Comment by laflannery** [comment]
Closing this in favor of a more recent/targeted issue (#13201)
